numba_dppy/dppl_passes.py

```diff
@@ -38,6 +40,80 @@ def dpnp_available():
     except:
         return False
 
+rewrite_function_name_map = {"sum": (["np"], "sum"),
+                             "eig": (["linalg"], "eig")}
+
+class RewriteOverloadedFunctions(object):
+    def __init__(self, state, rewrite_function_name_map=rewrite_function_name_map):
+        self.state = state
+        self.function_name_map = rewrite_function_name_map
+
```

Comment:
A comment is needed showing an example of the result of the rewrite, _i.e._ before and after versions of the IR.

Reply:
Comment added.